.ci/generate.sh

```diff
@@ -35,6 +35,15 @@ fi
 # move the generated CRD to the same location the operator-sdk places
 mv deploy/crds/jaegertracing.io_jaegers.yaml deploy/crds/jaegertracing.io_jaegers_crd.yaml
 
+# the controller-gen will generate a list of CRDs, but the operator-sdk tooling expects
+# a single item
+# the proper solutions are, in order:
+# 1) find a controller-gen switch that makes it write only one CRD. Such a switch doesn't exist yet: https://git.io/JvX5D
```

Comment:
It might be worth opening an issue in the controller-gen repo to let them know that this is causing problems.